app/views/govuk_publishing_components/components/_step_by_step_nav.html.erb

```diff
@@ -77,8 +77,10 @@
             <%
               in_substep = false
               options[:step_nav_content_id] = step_nav_content_id
+              options[:step_title] = step[:title]
```

Comment:
This `options` object gets passed to a function which builds all the `<li>` elements. Therefore I needed to add the `step_title` and `ga4_tracking` boolean to this object, so that I can build the `ga4-link` object on each list element that has a link.